roles/static_inventory/tasks/checkpoint.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: create static inventory dir
+  file:
+    path: "{{ inventory_path }}"
```

Comment:
I don't understand -- what manual steps does it require? If you use the defaults, it'll be just `./inventory` just like now, but if you say `ansible-playbook -i some-other-inventory` that will just work, too (generating `some-other-inventory/hosts`).

Using `inventory_dir` means that ansible will take care of everything and no configuration is necessary.